### PR TITLE
Replacing Federalist with cloud.gov Pages

### PR DESCRIPTION
**Pull request by hursey013**
## Changes proposed in this pull request:
Addresses https://github.com/cloud-gov/cg-site/issues/2197

- Simple text swaps (Federalist -> cloud.gov Pages) where appropriate
- Update support email and slack channels
- Unpublished the "How pages works" page
- Updated netlify cms config/docs to reflect new url
- Updated app screenshots

## Security Considerations
None

### CODE REVIEW

File: _pages/pages/documentation/supported-site-engines.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Pages supports Jekyll plugins, enabling any plugins in a site's `_plugins` direc
 
 Several dependencies are already available for use in the building environment. These include `ruby`, `python`, and `node.js`. You can write plugins that take advantage of these without needing a `Gemfile`.
 
-To see the exact configuration of the build environment, see the [build environment `Dockerfile`](https://github.com/18F/federalist-garden-build/blob/main/Dockerfile).
+To see the exact configuration of the build environment, see the [build environment `Dockerfile`](https://github.com/cloud-gov/pages-garden-build/blob/main/Dockerfile).
```
Comment (drewbo) [inline _pages/pages/documentation/supported-site-engines.md]: `pages-build-container`

**Review by drewbo — CHANGES_REQUESTED** [review]
One small change inline and other notes here. Not sure if we want to break any of these out into separate PRs but let me know if you want me to jump on any of these:

### Need new screenshots
- [x] /pages/documentation/
- [x] /pages/documentation/getting-started-with-jekyll/
- [ ] /pages/documentation/instructional-demos/
- [x] /pages/documentation/previews/
- [x] /pages/documentation/customization/#pages-default-404-page
- [x] /pages/documentation/env-vars-on-pages-builds/

### Broken links
- [ ] /pages/documentation/custom-headers/#examples
- [ ] /pages/documentation/how-pages-works/
   - This page also needs link/text updates throughout

### Other
- [x] Update /pages/documentation/access-permissions/#personal-access for getting Pages access
- [ ] Migrate mono repo example at /pages/documentation/monorepos-on-pages/
- [x] Switch to `npm run pages` in examples
- [x] Update templates at `_data/pages/templates.yml` (not positive where they are used)

**Comment by hursey013** [comment]
Appreciate the feedback @drewbo! Couple of notes:

## Need new screenshots
- /pages/documentation/instructional-demos/ - I think it would be great to update these videos, but in the meantime I think keeping the older Federalist videos around is ok, thoughts?

## Broken links
- /pages/documentation/custom-headers/#examples seems to be working for me?
- /pages/documentation/how-pages-works/ - I actually unpublished this because it seemed overly technical for our audience. If we want to keep that page, I can open a separate issue because a significant amount of updates

## Other
- Update /pages/documentation/access-permissions/#personal-access for getting Pages access. Updated this page to reflect the current state of auth/orgs
-  Migrate mono repo example at /pages/documentation/monorepos-on-pages/ - requested access so we can move this to `cloud-gov`

**Comment by drewbo** [comment]
Looks great @hursey013 
- Agree with waiting on updating the videos
- I didn't specify correctly but the broken link at `/pages/documentation/custom-headers/#examples` is the site linked right below (https://github.com/18F/federalist-uswds-gatsby2/blob/main/federalist.json)
- Let's open a separate issue to chat about the "how it works" page but agree with unpublishing for now
- 👍 on the last two

I think the only remaining change is to find a custom header example to use underneath /pages/documentation/custom-headers/#examples or removing for now

**Review by drewbo — APPROVED** [review]

**Pull request merged**